includes/razorpay-webhook.php

```diff
@@ -123,6 +133,20 @@ public function process()
                     return;
                 }
 
+                if (!get_post_meta($orderId, "rzp_webhook_notified_at", true))
+                {
+                    update_post_meta($orderId, "rzp_webhook_notified_at", time());
+                    error_log("webhook conflict due to early execution");
+                    return 1;
```

Comment:
Can we set the header with the proper status code and exit the script here? In a similar way which was implemented on Magento.

Reply:
When I am using header it is giving a timeout error and I have used return instead of exit because I wanted to differentiate between the scenarios when conflict is occurring and when it is not occurring.